packages/app/src/app/pages/Dashboard/Content/routes/MyContributions/index.tsx

```diff
@@ -51,6 +57,13 @@ export const MyContributionsPage = () => {
         showFilters={Boolean(param)}
         showSortOptions={Boolean(param)}
       />
+      {!isNotificationDismissed ? (
+        <Notification pageType={pageType}>
+          {itemsToShow().length === 0
```

Comment:
let's call `itemsToShow()` once at the top and use that result everywhere, I get performance anxiety when I see this, even though it's just a micro optimization :D